__tests__/app.test.js

```diff
@@ -87,4 +87,27 @@ describe("404: Not Found",()=>{
 
         return request(app).get('/api/topicss').expect(404)
     })
+})
+
+
+describe("GET /api/articles",()=>{
+
+    test("200: /api/articles should return an object with all current articles",()=>{
+        return request(app).get('/api/articles').expect(200)
+        .then(({body})=>{
+            expect(body.articles.length).toBeGreaterThan(0)
```

Comment:
Remember that we are dealing with test data and we are reseeding the db after each test - you can be more specific here. How many articles are there?